samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -671,7 +673,43 @@ static int clock_init(void)
 
 	return err;
 }
-#endif /* defined(CONFIG_CLOCK_CONTROL_NRF) */
+
+#elif defined(CONFIG_CLOCK_CONTROL_NRF2)
+
+int clock_init(void)
+{
+	int err;
+	int res;
+	const struct device *radio_clk_dev =
+		DEVICE_DT_GET_OR_NULL(DT_CLOCKS_CTLR(DT_NODELABEL(radio)));
+	struct onoff_client radio_cli;
+
+	/** Keep radio domain powered all the time to reduce latency. */
+	nrf_lrcconf_poweron_force_set(NRF_LRCCONF010, NRF_LRCCONF_POWER_DOMAIN_1, true);
+
+	sys_notify_init_spinwait(&radio_cli.notify);
+
+	err = nrf_clock_control_request(radio_clk_dev, NULL, &radio_cli);
+
+	do {
+		err = sys_notify_fetch_result(&radio_cli.notify, &res);
+		if (!err && res) {
+			printk("Clock could not be started: %d\n", res);
+			return res;
+		}
+	} while (err == -EAGAIN);
+
+#if defined(NRF54L15_XXAA)
+	/* MLTPAN-20 */
+	nrf_clock_task_trigger(NRF_CLOCK, NRF_CLOCK_TASK_PLLSTART);
+#endif /* defined(NRF54L15_XXAA) */
```

Comment:
NRF_CLOCK_TASK_PLLSTART task triggering applies to nRF54L15. 
Since CONFIG_CLOCK_CONTROL_NRF2 is for nRF54HX, it seems that this part is redundant.